options/locale/locale_en-US.ini

```diff
@@ -368,7 +368,7 @@ reset_password_mail_sent_prompt = A confirmation email has been sent to <b>%s</b
 active_your_account = Activate Your Account
 account_activated = Account has been activated
 prohibit_login = Sign In Prohibited
-prohibit_login_desc = Your account is prohibited to sign in, please contact your site administrator.
+prohibit_login_desc = Your account has been banned, please contact your site administrator for more information.
```

Comment:
That's not true, there are also other reasons why an account cannot sign in, i.e. it is a bot account or belongs to an organization.

Reply:
ACK